openwisp_network_topology/api/views.py

```diff
@@ -201,6 +206,8 @@ class LinkListCreateView(ProtectedAPIMixin, generics.ListCreateAPIView):
     ).order_by('-created')
     serializer_class = LinkSerializer
     pagination_class = ListViewPagination
+    filter_backends = (DjangoFilterBackend,)
+    filterset_fields = ('topology', 'organization', 'status')
```

Comment:
Please mention the filters in the README with a short example, applies to all other added filters, eg:

Available filters:

- `topology`, eg: `?topology=<topology_ID>`
- `status` eg: `?status=fetch`